src/Components/edit/Edit.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useRef } from "react";
+import axios from "axios";
+import { useLocation, useNavigate } from "react-router-dom";
+import { FormikState, useFormik } from "formik";
+import { useQuery, useMutation } from "react-query";
+import { Toast } from "primereact/toast";
+import { Employee } from "../../Models/Employee";
+
+export function Edit(): React.JSX.Element {
+  const { state } = useLocation();
+  const id = state.id;
+  const URI = `https://localhost:7189/api/employees/${id}`;
+
+  const navigate = useNavigate();
+
+  const toast = useRef<Toast>(null);
+
+  const getEmployee = useQuery({
+    queryKey: ["getEmployee"],
+    queryFn: () => {
+      axios
```

Comment:
I still do not see seggragation here for these requests.
What would be: 1. Components only to call mutations.
2. All mutation declaration to be extracted into dedicated file(s). Every mutation refer mutationFn: someFunctionWithAxios
3. all request are extracted into dedicated file(s)